base/cvd/cuttlefish/host/commands/cvd/server_command/status.cpp

```diff
@@ -35,8 +35,12 @@
 #include "host/libs/config/config_constants.h"
 
 namespace cuttlefish {
+namespace {
 
-static constexpr char kHelpMessage[] = R"(
+constexpr char kSummaryHelpText[] =
+    "Query and return status of currently running devices";
```

Comment:
This only returns status for a single instance group, it should probably direct people to use `cvd fleet` instead.

Reply:
Done